src/guide/introduction.md

```diff
@@ -200,7 +200,14 @@ If you are new to Vue, here's our general recommendation:
 
   - Go with Composition API + Single-File Components if you plan to build full applications with Vue.
 
-You don't have to commit to only one style during the learning phase. The rest of the documentation will provide code samples in both styles where applicable, and you can toggle between them at any time using the **API Preference switches** at the top of the left sidebar.
+You don't have to commit to only one style during the learning phase. The rest of the documentation will provide code samples in both styles where applicable, and you can toggle between them at any time using the **API Preference switches** at the top of the left sidebar. 
+
+> Use following shortcuts to switch between API Preference:
+> 
+> * `Ctrl+Alt+A` switches the API Preference (options/composition)  
```

Comment:
Small improvement here: we can use a dynamic text binding for the `Alt / Option` part and set it to the correct key after component mount using similar logic [here](https://github.com/vuejs/theme/commit/02522c09906d7f07d11f0cc3911459adbcd7fa73)

Reply:
navigator.platform is not available from navigator nor from globalThis/window (crash on vitepress build)

Reply:
Should be fixed now. I take the opportunity to separate things